README.md

```diff
@@ -97,7 +97,16 @@ This process takes sequenced entries and merges them into the log.
 Once this process has been completed, a new entry will:
  - Be available via the read API at the index that was returned from sequencing
  - Have Merkle tree hashes that commit to this data being included in the tree
- - Be committed to by the latest Checkpoint (and any Checkpoints issued after this point)
+
+### Publishing
+
+Publishing is a background process that creates a new Checkpoint for the latest tree.
+This background process runs periodically (configurable via [WithCheckpointInterval](https://pkg.go.dev/github.com/transparency-dev/trillian-tessera#AppendOptions.WithCheckpointInterval) and performs the following steps:
+  1. Create a new Checkpoint and sign it with the signer provided by [WithCheckpointSigner](https://pkg.go.dev/github.com/transparency-dev/trillian-tessera#AppendOptions.WithCheckpointSigner)
+  2. Contact witnesses and collect enough countersignatures to satisfy and witness policy configured by [WithWitnesses](https://pkg.go.dev/github.com/transparency-dev/trillian-tessera#AppendOptions.WithWitnesses)
+  3. If the witness policy is satisfied, make this new checkpoint public available
```

Comment:
nit: consistency of `checkpoint` vs `Checkpoint` above

Reply:
Gone with Big C.